packages/dashboards/addon/components/dashboard-filters.ts

```diff
@@ -28,7 +28,7 @@ export default class DashboardFiltersComponent extends Component<Args> {
   @tracked
   isAddingMode = false;
 
-  @computed('args.dashboard.filters.[]')
+  @computed('args.dashboard.filters.[]', 'store')
```

Comment:
maybe it's just me but it's unfortunate we have to add it here

Reply:
Yea some of the lints around computed don't seem to work very well so I figured I'll just fix it until we can move away from it